src/modules/sdk/objections/types.ts

```diff
@@ -14,8 +14,8 @@ export interface ApiError {
  */
 export enum ObjectionStatus {
   OPEN,
-  SUBMITTED,
   PROCESSED,
+  SUBMITTED,
```

Comment:
So this caused me some grief and confusion with its original ordering - I was setting ‘SUBMITTED’ in the web but the API was receiving and setting ‘PROCESSED’. Turns out that Javascript doesn’t really support enums and the typescript was generating code that meant what’s really set is an index value and not a string enum value. So… I set ‘SUBMITTED’ in the web which got sent to the API as ‘1’ (2nd item in web enum) which in turn got decoded in the API to ‘PROCESSED’ (as this is the 2nd item in the API ObjectionStatus enum). Upshot is that Javascript/Java enums duplicated between web and API need to have the same ordering. Hope this makes sense and you agree with this synopsis?

(BTW went back and checked the PTF enums and it seems we did also align the ordering there too)

Reply:
ah, what we need are sting enums then:
enum Direction {
    Up = "UP",
    Down = "DOWN",
    Left = "LEFT",
    Right = "RIGHT",
}
https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
Ah right. Will change and try this approach.

Reply:
Yep, that works. Updated.